src/xc_integrator/local_work_driver/host/reference_local_host_work_driver.cxx

```diff
@@ -141,7 +141,55 @@ namespace GauXC {
  
   }
   
-  
+  void ReferenceLocalHostWorkDriver::eval_uvvar_lda_gks( size_t npts, size_t nbe, const double* basis_eval,
+    const double* Xs, size_t ldxs, const double* Xz, size_t ldxz,
+    const double* Xx, size_t ldxx, const double* Xy, size_t ldxy, double* den_eval, double* K) {
+
+
+    auto *K2 = K; // KZ // store K in the Z matrix
+    auto *K3 = K2 + npts;
+    auto *K4 = K3 + npts;
+ 
+    for( int32_t i = 0; i < (int32_t)npts; ++i ) {
+
+      const size_t ioffs = size_t(i) * ldxs;
+      const size_t ioffz = size_t(i) * ldxz;
+      const size_t ioffx = size_t(i) * ldxx;
+      const size_t ioffy = size_t(i) * ldxy;
+
+      const auto*   Xs_i = Xs + ioffs;
+      const auto*   Xz_i = Xz + ioffz;
+      const auto*   Xx_i = Xx + ioffx;
+      const auto*   Xy_i = Xy + ioffy;
+
+      const double rhos = blas::dot( nbe, basis_eval + ioffs, 1, Xs_i, 1 );
+      const double rhoz = blas::dot( nbe, basis_eval + ioffz, 1, Xz_i, 1 );
+      const double rhox = blas::dot( nbe, basis_eval + ioffx, 1, Xx_i, 1 );
+      const double rhoy = blas::dot( nbe, basis_eval + ioffy, 1, Xy_i, 1 );
+ 
+      double mtemp = rhoz * rhoz + rhox * rhox + rhoy * rhoy;
+      double mnorm = 0;
+
+      if (mtemp > 1.0e-24) {
```

Comment:
I dislike "magic numbers". We should somehow make this configurable (preferably runtime, but I would accept a `constexpr` variable just so we can put a name on it

Reply:
Yep added in runtime configurability for this

Reply:
I see that, thanks! I've been meaning to add a set of RT parameters for XC, just hadn't gotten around to it! See updated review comment.